src/india_api/internal/service/resample.py

```diff
@@ -5,6 +5,8 @@
 
 
 def resample_generation(values: list[ActualPower], internal_minutes: int) -> list[ActualPower]:
+    if not values:
```

Comment:
i think this should `if len(values) == 0`

Reply:
fair enough, then stick with that. I find it harder to read, but its probably quicker